src/device/scrollCursorIntoView.ts

```diff
@@ -37,9 +38,10 @@ const scrollIntoViewIfNeeded = (el: Element | null | undefined) => {
   const y = window.scrollY + rect.y
 
   // leave a margin between the element and the viewport edge equal to half the element's height
+  // add offset to account for the navbar height and prevent scrolled to elements from being hidden below
   const scrollYNew = isAboveViewport
-    ? y - (toolbarRect?.height ?? 0) - rect.height / 2
-    : y - viewport.innerHeight + viewport.virtualKeyboardHeight + rect.height * 1.5
+    ? y - (toolbarRect?.height ?? 0) - rect.height / 2 + (navbarRect?.height ?? 0)
+    : y - viewport.innerHeight + viewport.virtualKeyboardHeight + rect.height * 1.5 + (navbarRect?.height ?? 0)
```

Comment:
I don't think it makes sense to add the offset when `isAboveViewport` is true. Wouldn't the NavBar affect the scroll position when the cursor is below the viewport?

Reply:
You're absolutely right, will fix.